data/builtins/thingengine.builtin/manifest.tt.in

```diff
@@ -196,9 +196,23 @@ class @org.thingpedia.builtin.thingengine.builtin
   #[confirm=false];
 
   action alert()
-  #_[result="Time's up!"]
+  #_[result="Alert"]
+  #_[formatted=[
+    { type="sound", name="alarm-clock-elapsed", exclusive=true }
+  ]]
   #_[canonical="alert"]
-  #[doc="makes Genie show/say a generic alert"]
+  #[doc="makes Genie show/play a generic alert"]
+  #[confirm=false];
+
+  action timer_expire(out message_output: String
+                      #_[canonical="message"])
+  #_[result="${message_output}"]
+  #_[formatted=[
+    { type="sound", name="alarm-clock-elapsed", exclusive=true },
+    { type="text", text="${message_output}"}
```

Comment:
You don't need a text item here. The `#_[result]` phrase will be spoken before the sound is played.